javascript-arrays/main.js

```diff
@@ -0,0 +1,24 @@
+const colors = ['red', 'white', 'blue'];
+console.log('Value of colors[0] ', colors[0]);
+console.log('Value of colors[1] ', colors[1]);
+console.log('Value of colors[2] ', colors[2]);
+
+const America = 'America is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.';
```

Comment:
Variable names should always start with lowercase! JavaScript uses camelCase. Also you're missing an `and`! Compare your output to the example output in the instructions.